internal/operatortest/envtest.go

```diff
@@ -92,6 +97,22 @@ func (ts *Suite) SetupSuite() {
 	ts.Env = testEnv
 	ts.Config = config
 	ts.Client = k8sClient
+
+	mgr, err := ctrl.NewManager(config, ctrl.Options{
+		MetricsBindAddress: "0",
+		Scheme:             ts.Scheme,
+		Port:               0,
+	})
+	ts.Require().NoError(err)
+	ts.Require().NotNil(mgr)
+	ts.Manager = mgr
+
+	ts.ExoClientMock = &ClientWithResponsesInterface{}
+
+	go func() {
+		err = mgr.Start(ts.Context)
+		ts.Require().NoError(err)
+	}()
```

Comment:
Why do we need a running manager in the background?

Reply:
if not started:

```
    logger.go:130: 2022-11-08T10:26:16.618+0100	DEBUG	Cannot get managed resource	{"controller": "testredis", "request": "/test-create", "error": "the cache is not started, can not read objects"}

```

the manager starts the runnables which start the cache, I guess that's what is missing otherwise. Didn't investigate more.

Reply:
ugh, ugly...
Can you add a comment for the reasoning? then LGTM

Reply:
indeed. Added a comment :+1: